database/migration/src/main/resources/changesets/embiggen_detritus_value.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="rmunshi" id="embiggen_call_caching_detritus_entry">
+        <modifyDataType tableName="CALL_CACHING_DETRITUS_ENTRY"
+                        columnName="DETRITUS_VALUE"
+                        newDataType="LONGTEXT"/>
```

Comment:
Some of the other files have a switch for the `clob` type so that different DBMSs get their own names for this datatype